### DISCUSSION TITLE
DAILY 15/05

### DISCUSSION THREAD

**Discussion by AhumadaCeleste** [AhumadaCeleste/DATA-WAVE#78] [General]
AYER 15/05: AYER POCO TIEMPO DESIGNADO AL PROYECTO.

HOY 16/05: TERMINAR CON LA RESPONSIBIDAD DE FORMULARIO Y LOGO.

MAÑANA: 17/05: VOY A AHACER LA CARGA D MATRICULA DEL INSTITUTO.

**Comment by cecipiombo** [comment]
Celeste no me coincide el día que tiene el título del Daily con lo escrito. Creo que este Daily es del 16/5

**Reply by AhumadaCeleste** [reply]
CORREGIDO
